### PR TITLE
Kjq/update deps

### PR DESCRIPTION
**Pull request by kentquirk**
@jebeck @jh-bate Ready for review. This was a little trickier than some of the others.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -102,42 +105,13 @@ var jsonp = function(response) {
...
 
   app.get('/status', function(request, response) {
-    response.send(200, 'OK');
+    response.status(200).send('OK');
   });
 
-  /*
-    used to process the carelink form data, now that is just for uploading and processing of the carelink csv
-  */
-  app.post(
```
Comment (jh-bate) [inline app.js]: so this is totally gone?

**Comment by jh-bate** [comment]
@kentquirk we are now missing the post to upload data for carelink 

http://localhost:9122/v1/device/upload/cl Failed to load resource: the server responded with a status of 404 (Not Found)

**Comment by jh-bate** [comment]
I have added back the carelink endpoint.

Another issue I am seeing an error on service restart 

e.g.

```tp_restart jellyfish```

```
{"name":"jellyfish","hostname":"Jamies-Air.fritz.box","pid":3642,"srcFile":"app.js","level":50,"err":{"message":"listen EADDRINUSE","name":"Error","stack":"Error: listen EADDRINUSE\n    at errnoException (net.js:904:11)\n    at Server._listen2 (net.js:1042:14)\n    at listen (net.js:1064:10)\n    at Server.listen (net.js:1138:5)\n    at Object.lifecycle.add.start (/Users/jhbate/Documents/Tidepool/src/jellyfish/app.js:286:43)\n    at Object.start (/Users/jhbate/Documents/Tidepool/src/jellyfish/node_modules/amoeba/lib/lifecycle.js:38:32)\n    at /Users/jhbate/Documents/Tidepool/src/jellyfish/app.js:321:13\n    at Object.<anonymous> (/Users/jhbate/Documents/Tidepool/src/jellyfish/app.js:324:3)\n    at Module._compile (module.js:456:26)\n    at Object.Module._extensions..js (module.js:474:10)","code":"EADDRINUSE"},"msg":"Uncaught exception bubbled all the way up!","time":"2015-03-23T23:53:45.791Z","v":0}
{"name":"hakken","hostname":"Jamies-Air.fritz.box","pid":3642,"src
```

**Comment by jh-bate** [comment]
yeah I thought it was a glitch also to start with so tried a few times with the same issue - then I asked @jebeck also and same issue there also

**Comment by kentquirk** [comment]
Ok, I'll look at it. Thanks.

On Mon, Mar 23, 2015 at 9:41 PM, Jamie <notifications@github.com> wrote:

> yeah I thought it was a glitch also to start with so tried a few times
> with the same issue - then I asked @jebeck <https://github.com/jebeck>
> also and same issue there also
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tidepool-org/jellyfish/pull/59#issuecomment-85285452>.
>



-- 
Kent Quirk
VP of Engineering, Tidepool

Tidepool is an open source, not-for-profit effort to build an open data
platform and better applications to reduce the burden of Type 1 Diabetes.

**Comment by jh-bate** [comment]
this has been replaced by PR #60

**Pull request closed without merging**